tests/std/tests/VSO_0000000_instantiate_containers/test.compile.pass.cpp

```diff
@@ -702,7 +702,6 @@ void extended_math_functions_test(T value) {
     (void) tanh(value);
 }
 
-// TRANSITION, VSO-251998

```

Comment:
I've reported this to the LWG and LEWG reflectors. I personally don't think it's worth the trouble to fix, but now I don't have to feel guilty about not doing so.